AdobeBranchExtension/src/main/java/io/branch/adobe/extension/AdobeBranchExtension.java

```diff
@@ -132,6 +145,34 @@ private void handleBranchConfigurationEvent(final Event event) {
         }
     }
 
+    /**
+     * Handle the Identity event by passing the Adobe ID to Branch
+     * @param event Adobe Event
+     */
+    private void handleIdentityEvent(final Event event) {
+        Map<String, Object> identitySharedState = getApi().getSharedEventState(ADOBE_IDENTITY_EVENT, event, this);
```

Comment:
Can this ever be empty but not null?  If so, the setting of `adobeIdHasBeenSet` might be misleading.